crypto/evp/e_chacha20_poly1305.c

```diff
@@ -232,7 +232,7 @@ static int chacha20_poly1305_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
             else if (len != plen + POLY1305_BLOCK_SIZE)
                 return -1;
 
-            if (ctx->encrypt) {                 /* plaintext */
+            if (EVP_CIPHER_CTX_encrypting(ctx)) {                 /* plaintext */
```

Comment:
Reindent the comment, I suggest reducing all spaces between `{` and the comment to one space